fitbert/fitb.py

```diff
@@ -12,13 +12,15 @@
 class FitBert:
     def __init__(
         self,
+        device=None,
```

Comment:
I want these arguments in order of descending-estimated-order-of-usage, but I really can't guess what they will be. Honestly, since there will probably be improvements to BERT in the near future (or [past](https://ai.facebook.com/blog/roberta-an-optimized-method-for-pretraining-self-supervised-nlp-systems/)), maybe we want `model_name` first? I think `device` will be near the end.